app/ui/src/app/customizations/api-connector/api-connector.service.ts

```diff
@@ -4,25 +4,28 @@ import { Observable } from 'rxjs/Observable';
 import { ApiHttpService } from '@syndesis/ui/platform';
 import { ApiConnectors, ApiConnectorData, CustomConnectorRequest } from './api-connector.models';
 
+
+import { apiConnectorEndpoints } from '../../customizations/api-connector/api-connector.api';
```

Comment:
This is fine, but you better use `import { apiConnectorEndpoints } from  './api-connector.api';` instead.

This way the service will keep pointing to the right file location even if we move the entire directory or change its name.